tensorflow_addons/optimizers/rectified_adam.py

```diff
@@ -87,15 +87,17 @@ def __init__(
         r"""Construct a new RAdam optimizer.
 
         Args:
-            learning_rate: A `Tensor` or a floating point value. or a schedule
-                that is a `tf.keras.optimizers.schedules.LearningRateSchedule`
+            learning_rate: A `Tensor` or a floating point value, or a schedule
+                that is a `tf.keras.optimizers.schedules.LearningRateSchedule`.
```

Comment:
The Dict is because tf.keras.optimizers.deserialize will pass a serialized config dict as learning rate or weight decay when using schedulers. It's not really intended for "regular" use. Do you have any suggestions about this?

Reply:
Got it, it makes sense to me.